python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -838,21 +838,145 @@ async def count_failed_verification_tests(
         # For at least one type of test, if the majority of tests fail in
         # the majority of detectors and the majority of exposures,
         # then don't certify the calibration
-        if failed_exposures_counter >= min_number_failed_exposures:
+        if failed_exposures_counter >= max_number_failed_exposures:
             certify_calib = False
 
         # Return a dictionary with the thresholds to report
         # them if verification fails.
         thresholds = {
-            "MIN_FAILURES_PER_DETECTOR_PER_TEST_TYPE_THRESHOLD": min_number_failures_per_detector_per_test,
-            "MIN_FAILED_DETECTORS_THRESHOLD": min_number_failed_detectors,
-            "MIN_FAILED_TESTS_PER_EXPOSURE_THRESHOLD": failure_threshold_exposure,
-            "MIN_FAILED_EXPOSURES_THRESHOLD": min_number_failed_exposures,
+            "MAX_FAILURES_PER_DETECTOR_PER_TEST_TYPE_THRESHOLD": max_number_failures_per_detector_per_test,
+            "MAX_FAILED_DETECTORS_THRESHOLD": max_number_failed_detectors,
+            "MAX_FAILED_TESTS_PER_EXPOSURE_THRESHOLD": failure_threshold_exposure,
+            "MAX_FAILED_EXPOSURES_THRESHOLD": max_number_failed_exposures,
             "FINAL_NUMBER_OF_FAILED_EXPOSURES": failed_exposures_counter,
         }
 
         return certify_calib, total_counter_failed_tests, thresholds
 
+    async def build_verification_report_summary(self, report_check_verify_stats):
+        """Helper function to print verification results.
+
+        Parameters
+        ----------
+        report_check_verify_stats : `dict`
+            Dictionary with verification results:
+
+        certify_calib : `bool`
+            Booolean indicating if the calibration should be certified
+            or not.
+
+        num_stat_errors : `dict`[`str`][`str`] or `None`
+            Dictionary with the total number of tests failed per exposure and
+            per cp_verify test type. If there are not any tests that failed,
+            `None` will be returned.
+
+        failure_thresholds : `dict`[`str`][`int`] or `None`
+            Dictionary reporting the different thresholds used to decide
+            whether a calibration should be certified or not (see `Notes`
+            below). If there are not any tests that failed,
+            `None` will be returned.
+
+        verify_stats : `dict`
+            Statistics from cp_verify.
+
+        Returns
+        -------
+        final_report_string : `str`
+            String with full report.
+        """
+
+        verify_report = report_check_verify_stats["NUM_STAT_ERRORS"]
+        thresholds_report = report_check_verify_stats["FAILURE_THRESHOLDS"]
+        verify_stats = report_check_verify_stats["VERIFY_STATS"]
+
+        final_report_string = ""
+        # List exposure IDs that have tests that failed
+        final_report_string += "|Exposures with verification tests that failed:\n"
```

Comment:
Oh, I see here and elsewhere you did `"| <text> \n"`, I think in order for this to work you need to do `"<text> \n|"`.